tests/system/test_sqlalchemy_bigquery.py

```diff
@@ -774,3 +774,27 @@ def test_unnest(engine, bigquery_dataset):
         f" unnest(`{bigquery_dataset}.test_unnest_1`.`objects`) AS `foo_objects`"
     )
     assert sorted(r[0] for r in conn.execute(query)) == ["a", "b", "c", "x", "y"]
+
+
+@pytest.mark.skipif(
+    packaging.version.parse(sqlalchemy.__version__) < packaging.version.parse("1.4"),
+    reason="regexp_match support requires version 1.4",
+)
+def test_regexp_match(session, table):
+    results = session.query(table.c.string).where(
+        table.c.string.regexp_match(".*52 St &.*").all()
+    )
+
+    assert len(results) == 12
+
+
+@pytest.mark.skipif(
+    packaging.version.parse(sqlalchemy.__version__) < packaging.version.parse("1.4"),
+    reason="regexp_match support requires version 1.4",
+)
+def test_not_regexp_match(session, table):
+    results = session.query(table.c.string).where(
+        not_(table.c.string.regexp_match("^Barrow St & Hudson St$"))
+    )
```

Comment:
Thinking this might do it.

```suggestion
def test_not_regexp_match(session, table):
    results = session.query(table.c.string).where(
        not_(table.c.string.regexp_match("^Barrow St & Hudson St$"))).all()
```